cloudmarker/events/azrdbmssslenforcementevent.py

```diff
@@ -0,0 +1,104 @@
+"""Microsoft Azure RDBMS SSL Enforcement Event.
+
+This module defines the :class:`AzRDBMSSLEnforcementEvent` class that
+identifies RDBMS servers which have SSL connection enforcement
+disabled. This plugin works on the  properties found in the ``ext``
+bucket of ``rdbms`` records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class AzRDBMSSSLEnforcementEvent:
+    """Az RDBMS SSL enforcement event plugin."""
```

Comment:
I like one of your suggestions on chat about renaming this plugin to `AzRDBMSEnforceSSLEvent`.

In fact, I also think that this can be a generic multi-cloud plugin that works for both Azure and GCP clouds. Whether TLS connectivity is enforced or not is a generic feature of all DBs in all clouds. There are only three possible answers: yes, no, or unknown which we can capture as `True`, `False`, or `None`. In that case the name `RDBMSEnforceSSL` or `RDBMSEnforceTLS` would be more suitable.

Reply:
Renamed the plugin to `RDBMSEnforceTLSEvent`